index.js

```diff
@@ -6,15 +6,29 @@ module.exports = {
   name: require('./package').name,
 
   setupPreprocessorRegistry(type, registry) {
+    if (type !== 'parent') {
+			return;
+		}
     let checker = new VersionChecker(this.project);
     let dep = checker.for('ember-source');
-    let maybePlugin = {
+    let version = dep.gte('3.17.0');
+		const plugin = this._buildPlugin({ version });
+    plugin.parallelBabel = {
+			requireFile: __filename,
+			buildUsing: '_buildPlugin',
+			params: options,
```

Comment:
`options` don't exist in this context.

```suggestion
			params:{},
```